flowey/flowey_cli/src/pipeline_resolver/ado_yaml.rs

```diff
@@ -511,15 +512,15 @@ EOF
         };
 
         ado_jobs.push(schema_ado_yaml::Job {
-            job: format!("job{}", job_idx.index()),
+            job: ado_job_name_overrides.get(&job_idx.index()).cloned().unwrap_or_else(|| format!("job{}", job_idx.index())),
```

Comment:
might as well break this expression out into its own variable, now that its not just a simple `format!` (which isn't too bad to repeat twice)